src/components/select/select.template.tsx

```diff
@@ -88,7 +88,8 @@ const SelectValue: FunctionalComponent<
         'mdc-floating-label--float-above':
             (!props.hasEmptyText && props.hasValue) ||
```

Comment:
The problem is here. `hasValue` should already work. But it obviously doesn't. We should check why `hasValue` is ignored for "multiple" type.

Reply:
fixed 🫡